Code/Framework/AzCore/AzCore/IO/Streamer/StorageDrive.cpp

```diff
@@ -60,9 +60,9 @@ namespace AZ::IO
         AZ_PROFILE_FUNCTION(AzCore);
         AZ_Assert(request, "PrepareRequest was provided a null request.");
 
-        if (AZStd::holds_alternative<FileRequest::ReadRequestData>(request->GetCommand()))
+            if (AZStd::holds_alternative<Requests::ReadRequestData>(request->GetCommand()))
         {
-            auto& readRequest = AZStd::get<FileRequest::ReadRequestData>(request->GetCommand());
+                auto& readRequest = AZStd::get<Requests::ReadRequestData>(request->GetCommand());
```

Comment:
nit: indentation looks off here I think?

Reply:
yes it is, will fix